plancton/__init__.py

```diff
@@ -263,7 +263,8 @@ def _create_container(self):
                            "Devices"     : [ dict(zip([ "PathOnHost", "PathInContainer",
                                                         "CgroupPermissions" ], x.split(":", 2)))
                                              for x in self.conf["devices"] ],
-                           "CapAdd"      : self.conf["capabilities"]
+                           "CapAdd"      : [ x.lstrip("+") for x in self.conf["capabilities"] if x and x[0]=="+" ]
+                           "CapDrop"     : [ x[1:] for x in self.conf["capabilities"] if x and x[0]=="-" ]
```

Comment:
I would say for consistency to use what you've actually did before (my bad, I keep changing my mind):

    [ x.lstrip("-") for x in self.conf["capabilities"] if x and x[0]=="-" ]